app/scripts/modules/core/src/pipeline/config/stages/stage.module.js

```diff
@@ -357,6 +357,12 @@ module(CORE_PIPELINE_CONFIG_STAGES_STAGE_MODULE, [
           body =
             '<p><strong>This pipeline is currently running - restarting this stage will result in multiple concurrently running pipelines.</strong></p>';
         }
+
+        if ($scope.execution.isRunning && $scope.execution.limitConcurrent) {
```

Comment:
does `scope.execution.isRunning` indicate that ANY execution is running? or that the _current execution_ is running?  Based on the wording I suspect it's the former, and that wouldn't be an accurate test.  It should be testing if any execution is running.

Reply:
Oh yes that is a good point, this only captures the case where a restart happens when the current execution is still running. I will go fix that to be any execution of this pipeline config :)